### PR TITLE
rating buckets in stats

### PR DESCRIPTION
**Pull request by kunchenguid**
this can't be merged yet. need to wait till we fix SS MMR records

### CODE REVIEW

File: packages/cloud/src/refreshCompetitiveStatsHandler.ts [hunk 1]
```diff
@@ -92,6 +91,34 @@ async function generateSpecStatsAsync() {
...
+                operation: 'GREATER_THAN_OR_EQUAL',
+                value: { int64Value: minRating.toFixed() },
+              },
+            },
+          },
+          {
+            filter: {
+              fieldName: 'customEvent:averageMMR',
+              numericFilter: {
+                operation: 'LESS_THAN',
```
Comment (prsauer) [inline packages/cloud/src/refreshCompetitiveStatsHandler.ts:117]: Should be LESS_THAN_OR_EQUAL?

File: packages/cloud/src/refreshCompetitiveStatsHandler.ts [hunk 2]
```diff
@@ -225,6 +246,34 @@ async function generateCompStatsAsync() {
...
```
Comment (prsauer) [inline packages/cloud/src/refreshCompetitiveStatsHandler.ts:272]: same as above

File: packages/cloud/src/refreshCompetitiveStatsHandler.ts [hunk 3]
```diff
@@ -320,6 +360,34 @@ async function generateCompStatsAsync() {
...
```
Comment (prsauer) [inline packages/cloud/src/refreshCompetitiveStatsHandler.ts:386]: same as above

**Comment by prsauer** [comment]
Is there a good way for us to preview the results of this against production data? Would be nice to spot check it before it goes out

File: packages/cloud/src/refreshCompetitiveStatsHandler.ts (hunk 1 above)
Comment (kunchenguid) [reply packages/cloud/src/refreshCompetitiveStatsHandler.ts]: good catch!

**Review by prsauer — APPROVED** [review]

**Pull request merged**